### ISSUE TITLE
Synology mediaserver_log_location /  Check: Log file does not exist

### ISSUE THREAD

**Issue by moreChalk** [jesseward/plex-lastfm-scrobbler#29]
I was wondering if anyone else had trouble with setting mediaserver_log_location in their config. 

**TL:DR** - I was able to correct the issue by pointing the  mediaserver_log_location config to a symlink, where the path had no space characters in it. 

On Synology Diskstations, the logs are stored at:
```sh
/volume1/Plex/Library/Application\ Support/Plex\ Media\ Server/Logs/Plex\ Media\ Server.log
```

When I set the file path in the config, the script could not verify the log file exists (and the script would error out upon run). I tried escaping the spaces with a backslash, and putting the path in both single and double quotes. All failed on the pre-check and at run. Verified this was correct location and that the user had permission to read the logs.

Tried the following configs:
```sh
mediaserver_log_location = /volume1/Plex/Library/Application\ Support/Plex\ Media\ Server/Logs/Plex\ Media\ Server.log
mediaserver_log_location = "/volume1/Plex/Library/Application Support/Plex Media Server/Logs/Plex Media Server.log"
mediaserver_log_location = '/volume1/Plex/Library/Application Support/Plex Media Server/Logs/Plex Media Server.log'
```
Fails pre-check in all three cases
```sh
(plex-lastfm-scrobbler)/volume1/homes/plex $ plex-scrobble.py -p
----------------------------------------------------------------------
| Running a few tests against your installation to ensure that
| required Plex Media Server files are in place.
----------------------------------------------------------------------
| * verify_plex_log_file_exists                          => [FAIL]
|       Check: Log file does not exist, please ensure you have
|       correctly set mediaserver_log location in your Plex Scrobbler
|       configuration.
| * detect_played_audio_in_PMS_log                       => [FAIL]
|       Check: Log file does not exist, please ensure you
|       havecorrectly set mediaserver_log location in your Plex
|       Scrobblerconfiguration.
| * was_lastfm_authorization_granted                     => [PASS]
----------------------------------------------------------------------
```

It was only after I created a symlink that didn't have a space to the path, that the pre-check and script were able to start scroobling correctly. 
```sh
(plex-lastfm-scrobbler)/volume1/homes/plex $ ln -s /volume1/Plex/Library/Application\ Support/Plex\ Media\ Server/Logs/Plex\ Media\ Server.log plexlog.log
```
```sh
mediaserver_log_location = /volume1/homes/plex/plexlog.log
```
```sh
(plex-lastfm-scrobbler)/volume1/homes/plex $ plex-scrobble.py -p
----------------------------------------------------------------------
| Running a few tests against your installation to ensure that
| required Plex Media Server files are in place.
----------------------------------------------------------------------
| * verify_plex_log_file_exists                          => [PASS]
| * detect_played_audio_in_PMS_log                       => [PASS]
| * was_lastfm_authorization_granted                     => [PASS]
----------------------------------------------------------------------
```

Has anyone else run into this issue?

**Comment by jesseward** [comment]
@moreChalk this has sat idle for some time, apologies for that. I am not really sure what to recommend.

If you have time to troubleshoot, i would be interested to see what happens without the symlink. Would you attempt to run the command under strace? For example
```
strace -o /tmp/plex.out -f python plex-scrobble.py
```
This will log all the system calls made during startup of plex-scrobble to /tmp/plex.out . Quick Google check hints that strace is available for Synology via the 'optware'  (ipkg?) packages, if not installed by default. 

** Disclaimer ** I am not familiar with Synology gear, or what possible negative impacts installing this package can bring. Please proceed at own risk ;)

**Comment by jesseward** [comment]
Closing out stale issue
